### ISSUE TITLE
Vill Q locks up all of my other programs.

### ISSUE THREAD

**Issue by tvdowntown** [aaron-elkins/Vill-Q-Issues#3]
Vill Q locks up all of my other programs when I try to use on my MacBook Pro, OS 10.10.5.  Once I open Vill Q, then no other application will work, everything freezes up. The only way to work with the computer is to do a complete shut down.

Please help.

Tony

**Comment by orklann** [comment]
Hi Tony,

I just replied your email with instructions to solve your issue, please try and let me know if it works for you. 

-Aaron

**Comment by tvdowntown** [comment]
Yes, I responded back.  I did as you suggested, but I still get locked up.

**Comment by tvdowntown** [comment]
Hi Aaron,

Are you still addressing issues with Vill Q?  Pleae let me know.

Tony

**Comment by orklann** [comment]
Hi, Tony,
I replied to you by email, and I want a screencast on how you use Vill Q, because I did not get any similar issues from Vill Q users, so I need more info on this issue.

**Comment by tvdowntown** [comment]
I think I figured out what to do.

I will keep you posted.

All the best,

Tony Vitale
